save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityGeneralInfoProps.kt

```diff
@@ -278,6 +319,11 @@ val vulnerabilityGeneralInfoProps: FC<VulnerabilityGeneralInfoProps> = FC { prop
                             +"${credit.name} <${credit.contact?.joinToString { it }}>"
                         }
                     }
+                    saveContributors.forEach { saveContributor ->
```

Comment:
why we need it as a duplicate? saveContributors already exist in `cosv.credits` and rendered as icons

Reply:
didnt know about it

Reply:
changed